src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -113,5 +113,10 @@ public class StorageConfiguration : ConfigurationCategory
         [DefaultValue(false)]
         [ConfigurationEntry("Storage.Dangerous.SkipChecksumValidationOnDatabaseLoading", ConfigurationEntryScope.ServerWideOnly)]
         public bool SkipChecksumValidationOnDatabaseLoading { get; set; }
+
+        [Description("EXPERT: Allows to load a database regardless encountered data integrity errors of already synced transactions in journals during the recovery operation on startup")]
+        [DefaultValue(false)]
```

Comment:
Shouldn't we always ignore this? Why make this optional?

Reply:
I was thinking about enabling it by default, but I don't like to silently ignore this kind of errors, even if that's safe. I prefer a user to make aware of such error. 

I propose that if we detect that a corrupted transaction was corrupted then we'll be thowing still, but we'll include to the exception message info it can be potentially fixed (without any data loss) by enabling this option. What do you think?

Reply:
We saw how that turned out with the missing journal. Even with the actual fix in the message , we got a lot of support calls and scared users.
Given that we *know* that this is an okay scenario, no data loss, why force it on the user?

Reply:
I think this case is slightly different because I believe that in this scenario users might be running into this due to hardware problem. So we need to let them know about that even if it's safe to continue the recovery operation.

I don't want to do it completely silently. I think this will be very rare BTW

Reply:
So let's make this an alert by default? I agree that we want the user to be aware of this.

Reply:
Okay. Let's alert them at least.

Reply:
Added in https://github.com/ravendb/ravendb/pull/9654/commits/6eb47e553813fbabcb6ccaadf0cdca75d266cbac